tsconfig.json

```diff
@@ -6,8 +6,7 @@
     "isolatedModules": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "target": "es6",
-
+    "target": "es2017",
```

Comment:
Reason?

Reply:
Oh, it was to allow root level await functionality, that I used for testing, will revert it.